ome/omero-parade
# support string columns table data

See https://github.com/ome/omero-parade/issues/47

This allows table data in String columns to be loaded.
To test:
 - Create table with string columns, e.g as described at https://github.com/ome/omero-parade/issues/47#issuecomment-414998542
 - In OMERO.parade, click ```Add table data...``` and choose a String column
 - Data should be loaded, displayed and is sortable etc

<img width="562" alt="Screen Shot 2019-03-11 at 03 42 31" src="https://user-images.githubusercontent.com/900055/54099208-f75e4900-43af-11e9-817b-ae4a26e99c0c.png">

This simply ignores any ```TypeError``` when generating histogram or min/max values for data.
At this point we don't know the column type, so we can't test for numerical columns (as we do for filtering at https://github.com/ome/omero-parade/blob/master/omero_parade/table_filters/omero_filters.py#L74)
I don't know what the intended usage of the ```min, max, and histogram``` JSON data was in the UI. It doesn't seem to be used currently, but there may be plans to use it?
cc @chris-allan @emilroz
